### ISSUE TITLE
model overfit

### ISSUE THREAD

**Issue by harishiker99** [harishiker99/IIC#1]
val_loss: 0.4416 >> training loss: 0.0319 after 20 epochs
